docs/sourcecode/particle-tracer.rst

```diff
@@ -0,0 +1,89 @@
+Particle Tracer
+===================
+
+This is an overview of the SedTRAILS particle tracer, which is designed to simulate the transport and dispersion of sediment particles in aquatic environments. The particle tracer is implemented as a Python package, and the source code is available in the `sedtrails.particle_tracer` module. 
+
+Data Retriever
+--------------
+The data retriever module is responsible for fetching and processing data required for particle tracking simulations. It includes functions to retrieve hydrodynamic data, sediment properties, and other relevant information needed for accurate particle transport modeling.
+
+.. automodule:: sedtrails.particle_tracer.data_retriever
+                    :members:
+
+
+Diffusion Library
+----------------
```

Comment:
Also, underscoring, for marking sections and subsections, must be at least as long as the text for the section. This is, underscoring can not undershoot, but it can overshoot. I this case, one `-` is missing 🫤